iocore/net/quic/QUICCongestionController.cc

```diff
@@ -62,6 +62,13 @@ QUICCongestionController::_in_recovery(ink_hrtime sent_time)
   return sent_time <= this->_recovery_start_time;
 }
 
+bool
+QUICCongestionController::is_app_limited()
+{
+  // FIXME : don't known how does app worked here
```

Comment:
not sure what we do here ?

https://tools.ietf.org/html/draft-ietf-quic-recovery-19#appendix-B.5

Reply:
Does "7.10.  Application Limited Sending" describe the behavior?
https://tools.ietf.org/html/draft-ietf-quic-recovery-19#section-7.10

Reply:
Generally, the congestion windows should not be changed in follow two:
 - The flow control limit the sending.
 - The pacer delay sending.

Currently we don't have pacer. And flow control is another case but out of this pr.  So could we fill an issue and fix that in another pr ?

Reply:
Thanks for filing an issue (#5468). I think this is enough for now.